src/network/networkctl-description.c

```diff
@@ -1,69 +1,44 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 
-#include "sd-bus.h"
+#include "sd-varlink.h"
 
 #include "alloc-util.h"
-#include "bus-error.h"
-#include "bus-locator.h"
-#include "bus-util.h"
 #include "glob-util.h"
 #include "json-util.h"
 #include "log.h"
-#include "networkctl.h"
 #include "networkctl-description.h"
 #include "networkctl-util.h"
+#include "networkctl.h"
```

Comment:
Not a matter, but @DaanDeMeyer prefers to locate `.` earlier than `-`? Either way, I do not think it is necessary to change the ordering of the header here. At least, it is unrelated to the change in this commit.